vision/segmentation/radius_clustering_voxel.go

```diff
@@ -21,10 +21,10 @@ type RadiusClusteringVoxelConfig struct {
 	MinPtsInPlane      int     `json:"min_points_in_plane"`
 	MinPtsInSegment    int     `json:"min_points_in_segment"`
 	ClusteringRadiusMm float64 `json:"clustering_radius_mm"`
-	WeightThresh       float64 `json:"weight_threshold"`
-	AngleThresh        float64 `json:"angle_threshold"` // in degrees
-	CosineThresh       float64 `json:"cosine_threshold"`
-	DistanceThresh     float64 `json:"distance_threshold"`
+	WeightThresh       float64 `json:"weight_threshold"` // TODO(RSDK-586): unit?
+	AngleThresh        float64 `json:"angle_threshold_degs"`
+	CosineThresh       float64 `json:"cosine_threshold"`   // TODO(RSDK-586): unit (rads)?
```

Comment:
WeightThresh is unitless
CosineThresh is unitless (it is between -1 and 1, the value after evaluating Cosine(theta) )
Distance is in mm